yarn.lock

```diff
@@ -81,36 +81,36 @@ __metadata:
   linkType: hard
 
 "@babel/core@npm:^7.1.0, @babel/core@npm:^7.12.3, @babel/core@npm:^7.16.0, @babel/core@npm:^7.7.5, @babel/core@npm:^7.8.4":
-  version: 7.19.3
-  resolution: "@babel/core@npm:7.19.3"
+  version: 7.19.6
```

Comment:
you don't need to modify the yarn.lock for this ticket, so just make sure not to include any changes in this file in the final PR